utils/generateHTMLs.js

```diff
@@ -0,0 +1,43 @@
+const klawSync = require('klaw-sync')
```

Comment:
Why is this all JS code anyway? That should all be implemented in TypeScript.

Reply:
I've turned my code to ts now.